servo/checks.py

```diff
@@ -31,6 +32,7 @@
 )
 
 import pydantic
+from tabulate import tabulate
 
 import servo.configuration
 import servo.logging
```

Comment:
add import for servo.events (note the event reduction may be better located in the servo if events import turns into a circular reference anywhere)